src/Products/Viewer/Entity/Web/LocalFileWrapper.cs

```diff
@@ -0,0 +1,103 @@
+﻿using GroupDocs.Viewer.MVC.Products.Common.Config;
+using GroupDocs.Viewer.MVC.Products.Common.Entity.Web;
+using GroupDocs.Viewer.MVC.Products.Common.Util.Comparator;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace GroupDocs.Viewer.MVC.Products.Viewer.Entity.Web
+{
+    public class LocalFileWrapper : IFileWrapper
+    {
+        private GlobalConfiguration globalConfiguration;
```

Comment:
Codacy found an issue: [Make 'globalConfiguration' 'readonly'.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Viewer-for-.NET-MVC/pullRequest?prid=6117782)